### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -30,6 +42,19 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 ---
 
+## @types/redux-mock-store
```
Comment (amyblais) [inline NOTICE.txt]: I'm not sure where to find this.

File: NOTICE.txt [hunk 2]
```diff
@@ -8,6 +8,18 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @formatjs/intl-pluralrules
```
Comment (amyblais) [inline NOTICE.txt]: I'm not sure where to find these two.

File: NOTICE.txt [hunk 3]
```diff
@@ -8,6 +8,37 @@ This document includes a list of open source components used in Mattermost Web A
...
```
Comment (amyblais) [inline NOTICE.txt]: Not sure if this is the right one.

File: NOTICE.txt [hunk 4]
```diff
@@ -8,6 +8,37 @@ This document includes a list of open source components used in Mattermost Web A
...
+ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+---
+
+## @formatjs/intl-relativetimeformat
+
+https://github.com/tc39/proposal-intl-relative-time
```
Comment (amyblais) [inline NOTICE.txt]: Not sure if this is the right one.

**Comment by grundleborg** [comment]
Adding @hmhealey on behalf of the webapp team as well.

File: NOTICE.txt [hunk 5]
```diff
@@ -8,6 +8,48 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @formatjs/intl-pluralrules
+
+This product contains '@formatjs/intl-pluralrules' by FormatJS.
+
+Polyfill for Intl.PluralRules
```
Comment (amyblais) [inline NOTICE.txt]: Not sure if the description is correct.

File: NOTICE.txt [hunk 6]
```diff
@@ -8,6 +8,48 @@ This document includes a list of open source components used in Mattermost Web A
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+---
+
+## @formatjs/intl-relativetimeformat
+
+This product contains '@formatjs/intl-pluralrules' by FormatJS.
+
+`Intl.RelativeTimeFormat`
```
Comment (amyblais) [inline NOTICE.txt]: Not sure if we can find a description for this.

File: NOTICE.txt (hunk 5 above)
Comment (hmhealey) [reply NOTICE.txt]: Yeah, that description is correct. `Intl.PluralRules` is something that the browser should provide, but some browsers don't support it yet.

File: NOTICE.txt (hunk 6 above)
Comment (hmhealey) [reply NOTICE.txt]: "Polyfill for Intl.RelativeTimeFormat" would work. It's a similar library to the other one.

**Pull request merged**